cmd/config-translator/translator.go

```diff
@@ -32,6 +32,8 @@ func initFlags() {
 	var inputMode = flag.String("mode", "ec2", "Please provide the mode, i.e. ec2, onPrem")
 	var inputConfig = flag.String("config", "", "Please provide the common-config file")
 	var multiConfig = flag.String("multi-config", "remove", "valid values: default, append, remove")
+	var strictValidation = flag.String("strict", "false", "Strict validation for agent json confil file. Valid values: true, false")
```

Comment:
We should use `flag.Bool` https://pkg.go.dev/flag#Bool so we don't accept arbitrary string values for this config

Reply:
True. Current fixing behind the scene with `make release`.

Reply:
On second thought, after using `flag.Bool`, there is certain behavior for it: we need to pass as `strict-validation=false` (only work with =); otherwise, it would always return true if we use flag strict-validation without equal sign. Do we want that to happen? Personally I would say no [since most of our documents start cwagent without equal side](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/install-CloudWatch-Agent-commandline-fleet.html). Therefore, would need to add some validations in the `config-translator` for this flag.